xml/deployment_cifs.xml

```diff
@@ -127,6 +127,25 @@
   oplocks = no
   kernel share modes = no
  </screen>
+     <para>
+      The above share configuration uses the <systemitem>vfs_ceph</systemitem>
+      module for &samba; to communicate with the &ceph; cluster. As a more
```

Comment:
I suggest

> as a higher-performance but less-flexible alternative

Reply:
My grammar isn't great, but shouldn't that be `a higher performing but less flexible alternative`?

Reply:
That's fine too. In all these instances, the _phrase_ is the adjective: 
* higher-performing
* higher-performance
* less-flexible

Thus the hyphens. Otherwise we would be talking about an alternative that is higher and is also performing, or which is less and is also flexible... which does not make sense.

Compare: three-year-old deer (1 deer, 3YO); three year-old deer (3 deer, all 1YO) three year old deer (does not make sense, as can be seen considering the adjectives separately: what does "3 year deer" mean?)

Reply:
I've pushed a new version with the hyphens added. Thanks